Sources/Apollo/ApolloStore.swift

```diff
@@ -252,6 +252,23 @@ public final class ApolloStore {
       try self.cache.removeRecord(for: key)
     }
 
+    /// Removes records with keys that match the specified pattern. This method will only
+    /// remove whole records, it does not perform cascading deletes. This means only the
+    /// records with matched keys will be removed, and not any references to them. Key
+    /// matching is case-insensitive.
+    ///
+    /// If you attempt to pass a cache key for a single field, this method will do nothing
+    /// since it won't be able to locate a record to remove based on that key.
+    ///
+    /// This method can be very slow depending on the number of records in the cache.
```

Comment:
Adding - Note: will make the generated docs look nicer.

```suggestion
    /// - Note: This method can be very slow depending on the number of records in the cache.
```